index.html

```diff
@@ -204,8 +204,64 @@ <h5 class="modal__headline modal-title" id="ModalLabel">Design.</h5>
     </div>
 </main>
 
-    <footer>
-        
+    <footer class="footer" id="contact">
```

Comment:
hmmmm,

twoj footer sklada sie z 2 sekcji:
1.) contact
2.) bottom footer

moze przenisiesz to id do kontenera ponizej (class="container"). Wiem, ze w przypadku tej kotwicy, ktora uzywasz w menu niewiele sie zmieni, ale bd to wygladalo profesjonalnie